pkg/plugins/policies/meshretry/api/v1alpha1/meshretry.go

```diff
@@ -0,0 +1,181 @@
+// +kubebuilder:object:generate=true
+package v1alpha1
+
+import (
+	k8s "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
+)
+
+// MeshRetry
+type MeshRetry struct {
+	// TargetRef is a reference to the resource the policy takes an effect on.
+	// The resource could be either a real store object or virtual resource
+	// defined inplace.
+	TargetRef common_api.TargetRef `json:"targetRef,omitempty"`
```

Comment:
why is it omitempty? Can this be empty?

Reply:
Right! I think initially we discussed having a syntactic sugar https://github.com/kumahq/kuma/blob/master/docs/madr/decisions/005-policy-matching.md?plain=1#L539. But this is not the case anymore, so I think I can update policy-generator and remove `omitempty` for all policies

Reply:
I created the issue to fix this https://github.com/kumahq/kuma/issues/5548